c/pthread-lit/assert.h

```diff
@@ -1,3 +1,6 @@
+#ifndef SVC_ASSERT_H
+#define SVC_ASSERT_H
```

Comment:
Why add this header guard? As far as I see there is no duplicate inclusion (and neither would it be problematic if the header got included twice).

Reply:
If the header got included twice, it would be a problem as the definition of `__VERIFIER_assert` would violate one definition rule. On Linux platforms this would result in a linker error. I agree that the file is included only once, but is is a good practice to prevent accidental double inclusion if the header does not work in this mode.